src/mesh.cpp

```diff
@@ -2999,6 +3003,21 @@ void LibMesh::initialize()
   auto first_elem = *m_->elements_begin();
   first_element_id_ = first_elem->id();
 
+  // if the mesh is adaptive elements aren't guaranteed by libMesh to be
+  // contiguous in memory, so we need to map from bin indices (defined over
+  // active elements) to global dof ids
```

Comment:
```suggestion
  // if the mesh is adaptive elements aren't guaranteed by libMesh to be
  // contiguous in ID space, so we need to map from bin indices (defined over
  // active elements) to global dof ids
```

Slight tweak to the comment here. Correct me if I'm misunderstanding the issue here though @nuclearkevin.